utils.js

```diff
@@ -1,13 +1,17 @@
 async function getSelfUser() {
-  const res = await fetch(`${API_BASE_URL}/users/self`, {
-    method: 'GET',
-    credentials: 'include',
-    headers: {
-      'Content-type': 'application/json',
-    },
-  });
-  const self_user = await res.json();
-  return self_user;
+  try {
+    const res = await fetch(`${API_BASE_URL}/users/self`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+    const self_user = await res.json();
+    if (res.status === 200) {
+      return self_user;
+    }
+  } catch (err) {}
```

Comment:
we can console the error here instead of leaving the catch block empty.

Reply:
it is not necessary to show the error info to user.